### ISSUE TITLE
[BUG] `crc console` for the microshift preset open file explorer on windows

### ISSUE THREAD

**Issue by themr0c** [crc-org/crc#3873]
### General information

  * OS: Windows 10 Pro VM running in libvirt.
  * Hypervisor: Hyper-V
  * Did you run `crc setup` before starting it (Yes/No)? No at the CLI. Installed via Podman Desktop extensions.
  * Running CRC on: VM

## CRC version
```bash
2.18
```
  
## CRC status
```bash
# Put `crc status --log-level debug` output here
```

## CRC config
```bash
# Put `crc config view` output here
```

## Host Operating System
```bash
# Put the output of `cat /etc/os-release` in case of Linux
# put the output of `sw_vers` in case of Mac
# Put the output of `systeminfo` in case of Windows
```

### Steps to reproduce

  1. Install Podman Desktop
  2. Install the OpenShift Local extension for Podman Desktop 
  3. Initialize a MicroShift instance
  4. `crc console` is opening the file explorer

### Expected

Opens the MicrShift console in the default web browser (or an error message if MicroShift does not support this feature)

### Actual

Opens the Windows file explorer

### Logs

Before gather the logs try following if that fix your issue
```bash
$ crc delete -f
$ crc cleanup
$ crc setup
$ crc start --log-level debug
```

Please consider posting the output of `crc start --log-level debug`  on http://gist.github.com/ and post the link in the issue.

**Comment by gbraad** [comment]
@anjannath I will test this, but can you also?
I see the same happening, which shouldn't

Low priority
